homeassistant/components/comelit/coordinator.py

```diff
@@ -47,3 +48,16 @@ async def _async_update_data(self) -> dict[str, Any]:
         await self.api.logout()
 
         return devices_data
+
+    def device_info(self) -> DeviceInfo:
```

Comment:
One thing I notice is that we change the identifiers of the device. And we merge all devices in one device. Are the lights separate devices you connect to one bridge comparable to Philips Hue?

Reply:
Nope, the bridge is the only device on the network with some extensions to get lights, covers, etc. 
I requested Erick review because we spoke about this change and agreed is the best approach.

Simone